lib/elliptic/curve/mont.js

```diff
@@ -147,6 +158,10 @@ Point.prototype.mulAdd = function mulAdd() {
   throw new Error('Not supported on Montgomery curve');
 };
 
+Point.prototype.eq = function eq(other) {
+  return this.getX().cmp(other.getX()) === 0;
```

Comment:
Its funny for you, but it scares me a lot. You patched this code all over, and didn't see a small function below. Now I will need to recheck everything :)